activity/activity.go

```diff
@@ -19,19 +19,21 @@ type Activity interface {
 type Factory func(ctx InitContext) (Activity, error)
 
 type InitContext interface {
-	
+
 	// Settings
 	Settings() map[string]interface{}
-	
+
 	// MapperFactory gets the mapper factory associated with the activity host
 	MapperFactory() mapper.Factory
-	
+
 	// Logger logger to using during initialization, activity implementations should not
 	// keep a reference to this
 	Logger() log.Logger
-	
-	// Name returns name of the activity
+
+	// Name returns name of the activity / task
 	Name() string
+	// HostName returns name of the flow / task host
+	HostName() string
```

Comment:
naming the property hostname is a little confusing, no?

Reply:
@mellistibco It is consistent with Context interface.